pages/concepts-limit-order-book-and-matching.md

```diff
@@ -0,0 +1,86 @@
+# Limit Order Book and Matching
+
+This document outlines the key differences between centralized exchanges and dYdX Chain, focusing on the decentralized limit order book and matching engine.
+
+
+## Blockchain Overview
+<!-- TODO: Move this to a general explainer section and link to it -->
+
+dYdX Chain is a p2p blockchain network using [CosmosSDK](https://github.com/cosmos/cosmos-sdk) and [CometBFT](https://github.com/cometbft/cometbft) (formerly Tendermint).
+
+Anyone can run a full node using the open-source software. Full nodes with sufficient delegated governance tokens participate in block building as validators.
+
+The software repository is: https://github.com/dydxprotocol/v4-chain/
+
+## Limit Order Book
+Each full node in the network maintains an in-memory order book, which undergoes state changes in real time as traders submit order instructions.
+
+Block proposers use trades from their local order book to build blocks, with matches generated by price-time priority. Since message arrival order varies between nodes, the order book may differ across the network at any given point in time. To address this, nodes partially sync their local books with each new consensus-committed block.
```

Comment:
The last sentence is a bit clunky to me. What does partially sync mean? 

It might be useful to add the word optimistic here to start denoting matches/orders into two types, optimistic and confirmed.

Reply:
I want to say something like "nodes do a [thing we'll explain later] to make their local books logically consistent with the block" without getting into the details of what they do or what logical consistency means.

I started with just "sync", but then realized that made it sound like each time there's a new block, nodes somehow gets their local books into identical states

How would you phrase it without going into details on the process?

Reply:
to keep it as simple as possible, i would say this for the last sentence:

When each new consensus-committed block is received, the proposed changes are committed. Afterwards, each node syncs their local orderbook on top of the new block's state in order to merge the in-consensus orderbook with the node's local orderbook.